docs/getting-started.md

```diff
@@ -52,16 +52,14 @@ const account = 'XXXXXXXXXXXXX';
 const region = 'us-east-2';
 
 const addOns: Array<blueprints.ClusterAddOn> = [
-    new blueprints.addons.ArgoCDAddOn,
-    new blueprints.addons.CalicoAddOn,
-    new blueprints.addons.MetricsServerAddOn,
-    new blueprints.addons.ClusterAutoScalerAddOn,
-    new blueprints.addons.ContainerInsightsAddOn,
+    new blueprints.addons.ArgoCDAddOn(),
+    new blueprints.addons.CalicoOperatorAddOn(),
+    new blueprints.addons.MetricsServerAddOn(),
+    new blueprints.addons.ClusterAutoScalerAddOn(),
     new blueprints.addons.AwsLoadBalancerControllerAddOn(),
     new blueprints.addons.VpcCniAddOn(),
     new blueprints.addons.CoreDnsAddOn(),
-    new blueprints.addons.KubeProxyAddOn(),
-    new blueprints.addons.XrayAddOn()
+    new blueprints.addons.KubeProxyAddOn()
```

Comment:
We are removing Xray which may make people think how i can do tracing now. Can we introduce ADOT, AMP, XRAY in Getting started ?

Reply:
I did not include XRayAdotAddOn here, because it has dependency on cert manager and adot and requires to test the example first. Xray adot has been deprecated in favor of the adot add-on. You can see the update to the XRay addon docs. 

This is just an example of a blueprint, there is no goal to include all possible addons.

Reply:
SGTM.